lib/python/rose/suite_engine_procs/cylc.py

```diff
@@ -115,7 +115,10 @@ def cmp_suite_conf(
             self.popen.run_simple("cylc", "register", suite_name, suite_dir)
         f_desc, new_suite_rc_processed = mkstemp()
         os.close(f_desc)
-        command = ["cylc", "validate", "-v", "-o", new_suite_rc_processed]
+        command = ["cylc", "validate"]
+        if debug_mode:
+            command += ['-v']
```

Comment:
Can probably combine with `if debug_mode:` branch a few lines below?

Reply:
Good point, we don't need verbose any more.